src/foam/nanos/ruler/RulerDAO.js

```diff
@@ -310,6 +310,8 @@ for ( Object key : groups.getGroupKeys() ) {
       setX(x);
       setDelegate(delegate);
       setDaoKey(serviceName);
+      // This doesn't get called when using Builder,
```

Comment:
lol i meant put this in capable when we call the constructor but I guess this works too